src/main/kotlin/nexters/admin/repository/AdminCacheRepository.kt

```diff
@@ -0,0 +1,26 @@
+package nexters.admin.repository
+
+import nexters.admin.domain.user.administrator.Administrator
+import org.springframework.stereotype.Repository
+import java.util.concurrent.ConcurrentHashMap
+
+@Repository
+class AdminCacheRepository(
+        private val adminRepository: AdministratorRepository,
+) {
+    private var cacheRepository: ConcurrentHashMap<String, Administrator> = ConcurrentHashMap()
+
+    fun findByUsername(username: String): Administrator? {
+        if (!cacheRepository.containsKey(username)) {
+            updateCache(username)
+        }
+        return cacheRepository[username]
+    }
+
+    private fun updateCache(username: String) {
+        val admin = adminRepository.findByUsername(username)
+        if (admin != null) {
+            cacheRepository[username] = admin
+        }
```

Comment:
취향차이지만, scope function을 이용하여 보다 kotlin 스럽게 작성할 수 있어보여요
```suggestion
        adminRepository.findByUsername(username)
                ?.let { cacheRepository[username] = it }
```

Reply:
반영했습니다.